apps/tlon-web/src/state/lure/lure.ts

```diff
@@ -211,29 +194,14 @@ export const useLureState = create<LureState>(
 );
 
 function getLureMetadata(flag: string, meta: GroupMeta, profile: Contact) {
-  const title = `Join ${meta.title || flag}`;
-  const description = meta.description || '';
-  const image = meta.cover || meta.image || undefined;
   const iconIsColor = meta.image ? meta.image.startsWith('#') : false;
-
   return {
-    $og_title: title,
-    $og_description: description,
-    $og_image_url: image,
-    $twitter_title: title,
-    $twitter_description: description,
-    $twitter_image_url: image,
-    $twitter_card: meta.cover
-      ? 'summary_large_image'
-      : meta.image
-        ? 'summary'
-        : undefined,
```

Comment:
why don't we need these?

Reply:
So on `tlon.network` these aren't used. For Branch, my thinking is we wanna have the service set them itself so we can control it in a single place. Notably, it already does that for personal invites